app/controllers/concerns/fields.rb

```diff
@@ -0,0 +1,11 @@
+module Fields
+  private
+
+  def field_params
+    return {} unless params[:fields]
+
+    params[:fields].transform_values do |field_value|
+      field_value.split(",")
+    end.to_unsafe_h
```

Comment:
This might be more of a code style thing, but it makes more sense to me to call `to_unsafe_h` on the `params` response and then transform it.

```suggestion
    params[:fields].to_unsafe_h.transform_values do |field_value|
      field_value.split(",")
    end
```

Reply:
it's fun how code style evolves over time, the positioning of `to_unsafe_h` is the same as in MPDX where you committed it 3 years ago ;)

https://github.com/CruGlobal/mpdx_api/blame/master/app/controllers/concerns/fields.rb#L9

Reply:
I blame Tatai for not reviewing that massive PR more closely ;)